deployer/tests/context.py

```diff
@@ -3,4 +3,4 @@
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
 import auth
-import deploy
+import deployer
```

Comment:
Since we don't have a hub fixture anymore and  the deployer is in it's own module now, this entire file can be deleted.

Reply:
Ahh, thanks @GeorgianaElena. done